synapse/storage/state.py

```diff
@@ -579,7 +584,7 @@ async def get_state_groups_ids(
         if not event_ids:
             return {}
 
-        event_to_groups = await self.stores.main._get_state_group_for_events(event_ids)
+        event_to_groups = await self._get_state_group_for_events(event_ids)
```

Comment:
`synapse/storage/persist_events.py` has two calls to `DataStore._get_state_group_for_events`. Do we need to update those too?